src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+const list = document.querySelectorAll('li');
+
+for (const item of list) {
+  if (item.children.length > 0 && item.firstElementChild.tagName === 'UL') {
```

Comment:
I think this check is unnecessary here

Reply:
without this statement every element after click event will be closed, include all `li's`, cause every item will contain `span` tag.

Reply:
okay, now I see
Another approach to this task would be to find all 'ul' elements and then cover in span only li tags which serve as a parent for ul list
in this case, you won't need to check every li element in your document ;)